data/tools/alex.yml

```diff
@@ -0,0 +1,10 @@
+name: alex
+category:
+  - linter
+tags:
+  - writing
+license: MIT License
+type: []
+source: 'https://github.com/get-alex/alex'
+homepage: 'https://alexjs.com/'
```

Comment:
Now that we're at it, would you mind standardizing the URLs with your magic skills? ✨ Maybe we can remove the trailing slash in the end - or even better: run it through a URL parser to have a consistent format.